app/views/payments/_form.html.slim

```diff
@@ -5,6 +5,7 @@
       = f.input :amount, input_html: { class: 'w3-input w3-border', autofocus: true, min: 0.01 }
       = f.input :date, as: :date, html5: true, input_html: { class: 'w3-input w3-border' }.merge!(form_params[:date])
       = f.association :category, { collection: current_user.all_categories, input_html: { class: 'w3-select w3-border'} }.merge!(form_params[:category])
+      = f.input :notes, input_html: { class: 'w3-input w3-border', rows: 2 }, as: :text
```

Comment:
Could you please make this field separately. Like in this screenshot 
![screenshot from 2016-06-28 17-20-24](https://cloud.githubusercontent.com/assets/14985448/16419081/16adc704-3d55-11e6-9fd0-89ee2f63d94f.png)